lib/tests/ruby/ruby.cc

```diff
@@ -528,7 +528,7 @@ SCENARIO("custom facts written in Ruby") {
         }
     }
     GIVEN("a custom on_message block") {
-        log_capture capture(level::debug);
+        log_capture capture(level::warning);
```

Comment:
This test was failing because Leatherman emits a debug message when file_util::file_readable emits a debug message when the file does not exist, as happens when it attempt to load the default config file, but there isn't one. Does this change still test what it was intended to test? It looks like we are doing the same thing to both debug and warn in the ruby fixture: https://github.com/puppetlabs/facter/blob/master/lib/tests/fixtures/ruby/on_message.rb.